examples/poem-openapi/README.md

```diff
@@ -0,0 +1,24 @@
+# poem-openapi with passage_flex
+
+This is an example of how to use the `passage_flex` crate with an HTTP server implemented with [poem-openapi](https://crates.io/crates/poem-openapi) in Rust.
+
+
+## How to run
```

Comment:
could also be good to point out that they would need to set their Authentication Origin in the Console app settings to match this example app's or else it could be difficult for them to troubleshoot the mismatching origins error

Reply:
this comment was referring to an issue i had with running the example, but on a second attempt it doesn't seem to be an issue